python/ray/_private/custom_types.py

```diff
@@ -118,6 +119,14 @@
 # and any modifications must be backward compatible.
 LANGUAGE = ["PYTHON", "JAVA", "CPP"]
 
+# The TensorTransport is used to define the tensor transport type.
```

Comment:
This comment is not very descriptive.

Reply:
Point to `common.proto` as the single source of truth.